README.md

```diff
@@ -36,6 +43,14 @@ Return to your original locations.  In this step all groups will act like Design
 
 ***Q1.3 Briefly describe your solutions (use a list) and any strengths/weaknesses you see.***
 
+1. Smart Table: A pad in the center that you put your cup on and alerts the bar/waiter that a cup needs to be refilled. Would also have a screen for drink customization.
```

Comment:
That's an intriguing idea. I imagine the waiters/waitresses and other staff would be able to view a map of smart tables where tables with empty drinks appear as red while tables without needs are green.